lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -1,37 +1,56 @@
+tgAttribute = (attr) ->
+  tgAttr = if attr[0...3] == 'tg-'
+    "data-#{attr}"
+  else
+    "data-tg-#{attr}"
+
+getTGAttribute = (node, attr) ->
+  tgAttr = tgAttribute(attr)
+  node.getAttribute(tgAttr) || node.getAttribute(attr)
+
+removeTGAttribute = (node, attr) ->
+  tgAttr = tgAttribute(attr)
+  node.removeAttribute(tgAttr)
+  node.removeAttribute(tgAttr)
+
+tgAttributeExists = (node, attr) ->
```

Comment:
I don't think this method is needed. We can just call `getTGAttribute(node, attr)?`